app/scenes/Login/index.tsx

```diff
@@ -66,6 +67,20 @@ function Login() {
     auth.fetchConfig();
   }, [auth]);
 
+  React.useEffect(() => {
+    async function load() {
+      if (auth.authenticated) {
+        await auth.fetch();
```

Comment:
Do you remember the need for this? Definitely a red flag when already authenticated

Reply:
`auth.team` was  `undefined` even if we are authenticated.

Reply:
<img width="695" alt="Screenshot 2022-02-09 at 3 06 01 AM" src="https://user-images.githubusercontent.com/40148448/153119841-be772067-d495-472d-bd8b-7720034391d5.png">
The Auth Store is empty in the local storage. It should have the team and user when logged in, right?

Reply:
It should yea, looks like there is a bigger bug here 😢  will take a look

Reply:
https://github.com/outline/outline/pull/3078

Reply:
Okay, fix merged - lets see if that helps tidy this up – this turned out to be a great find!